app/jobs/scheduled/delete_rejected_emails.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Jobs
+  class DeleteRejectedEmails < ::Jobs::Scheduled
+    every 1.month
```

Comment:
I would just add this to the weekly task. I don't think it needs a dedicate task.